src/domain/innovationHub/InnovationHubCardHorizontal/InnovationHubCardHorizontal.tsx

```diff
@@ -62,8 +63,13 @@ const InnovationHubCardHorizontal = ({
   profile: { displayName, description, url, banner },
   ...spaces
 }: InnovationHubCardHorizontalProps) => {
+  const { user: { hasPlatformPrivilege } = {} } = useUserContext();
+  const isPlatformAdmin = hasPlatformPrivilege?.(AuthorizationPrivilege.PlatformAdmin);
+
+  const SelectComponent = () => (url && isPlatformAdmin ? { component: RouterLink, to: url } : { component: Box });
```

Comment:
this is more of a `getComponentProps` function. It could be a component if you pass it to the component prop instead of spread.

Reply:
totally right, I forgot to clean this up, it doesn't even need to be a function or to return anything if it's not a link it's just a Box already